client/components/organisms/PageLayout/PageLayout.js

```diff
@@ -4,19 +4,46 @@ import treesIcon from "../../../assets/icons/trees.svg";
 import Container from "react-bulma-companion/lib/Container";
 import Burger from "../../atoms/Burger/Burger";
 import Menu from "../Menu";
+import AdminMenu from "../AdminMenu/AdminMenu";
+import AdminMenuSearch from "../AdminMenuSearch/AdminMenuSearch";
 
 export default function PageLayout(props) {
   const [open, setOpen] = useState(false);
+  const [admin, setAdmin] = useState(true);
```

Comment:
Now that we have the state of the user, let's use it to infer if the user is admin or not. (instead of a constant "true").